lib/API/JSON/ResponseBuilders/SubscribersResponseBuilder.php

```diff
@@ -69,12 +69,12 @@ private function buildListingItem(SubscriberEntity $subscriber): array {
     ];
   }
 
-  public function build(SubscriberEntity $subscriberEntity): array {
+  public function build(SubscriberEntity $subscriberEntity, bool $isMPAPI = false): array {
```

Comment:
Do you think we should use a different approach than the boolean argument? They are generally considered a [bad practice](https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3). 
But also if some other part of the code would require the extra data they would have to pass the argument `isMPAPI` even though it is a completely different part.

Do we even have to use the argument? What if we included the extra data always. I know it would consume a bit more memory, but I don't think it would be causing any problems. What do you think?

Reply:
I like the idea of always sending the extra data, I guess I was being too cautious about not causing regressions in other parts of the plugin. I will make the change.